src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -548,7 +661,7 @@ export class NotebookModel extends Disposable implements INotebookModel {
 		}
 		// Set newly created cell as active cell
 		this.updateActiveCell(cell);
```

Comment:
Does removing this line break functionality when adding a new cell (without splitting)? We need to ensure that users can type into a newly-created cell without any further interactions

Reply:
The inserted cell need to be made active so that it is in editable state. Verified that user can type into newly created cell without splitting